src/ray/core_worker/context.h

```diff
@@ -117,6 +117,10 @@ class WorkerContext {
 
   bool CurrentActorIsAsync() const ABSL_LOCKS_EXCLUDED(mutex_);
 
+  void SetCurrentActorShouldExit() ABSL_LOCKS_EXCLUDED(mutex_);
+
+  bool GetCurrentActorShouldExit() const ABSL_LOCKS_EXCLUDED(mutex_);
```

Comment:
Please add header comments for these indicating what they're used for

Reply:
fixed

Reply:
Added 3 tests: `test_exit_actor_normal_actor_in_constructor_should_exit`, `test_exit_actor_async_actor_in_constructor_should_exit`, and `test_exit_actor_async_actor_nested_task_in_constructor_should_exit`